app/controllers/pusher_controller.rb

```diff
@@ -4,10 +4,27 @@ class PusherController < ApplicationController
   # stop rails CSRF protection for this action
   protect_from_forgery except: :auth
 
+  # POST /pusher/auth
+  # @argument channel_name [String] The name of the pusher channel.
+  # @argument socket_id [String] The id of the pusher socket.
+  #
+  # Returns an auth token from Pusher that looks like this
+  # {"auth": ":7944d5b3dc2915f6a9c55dbedcbe91037799bb3627fc868890c2a7fd6f5fde4b"}
```

Comment:
Is this an actual Pusher auth key that's in use? If so, we should remove it from this comment and generate a new one. You could replace the string with `"some-auth-key"`.

Reply:
Hey Julia! No, this isn't a real one. We just put it up as an example, but I'll swap it out for "some-auth-key" instead so others don't worry too :)

Thanks!